pkg-manager/core/src/install/index.ts

```diff
@@ -320,7 +320,14 @@ export async function mutateModules (
     ignoredBuilds: result.ignoredBuilds,
   }
 
-  async function _install (): Promise<{ updatedProjects: UpdatedProject[], stats?: InstallationResultStats, depsRequiringBuild?: DepPath[], ignoredBuilds: string[] | undefined }> {
+  interface InnerInstallResult {
+    readonly updatedProjects: UpdatedProject[]
+    readonly stats?: InstallationResultStats
+    readonly depsRequiringBuild?: DepPath[]
+    readonly ignoredBuilds: string[] | undefined
+  }
```

Comment:
Defined the inline interface here so I could reuse it in the new `tryFrozenInstall()` function.